scripts/configure.sh

```diff
@@ -360,45 +365,47 @@ ${build_cmd}_function () {
-    }
-    source ~/.bashrc
-
-    shell="$SHELL"
-    if [[ ${shell}  == *"zsh"* ]]; then
-        echo "Setting .zshrc with environment variables..."
-        grep -qF 'source /opt/ros/'$ros_version'/setup.zsh' ~/.zshrc || echo 'source /opt/ros/'$ros_version'/setup.zsh' >> ~/.zshrc
-        grep -qF ${cmake_astrobee_path} ~/.zshrc || {
-            echo -e '\n' >> ~/.zshrc
-            echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.zshrc
-        }
-    fi
 
-    catkin profile add ${profile:-native}
-    catkin profile set ${profile:-native}
-    catkin config --no-extend \
-        --build-space ${workspace_path}build \
-        --install-space ${install_path:-${workspace_path}install} \
-        --devel-space ${workspace_path}devel \
-        --log-space ${workspace_path}log \
-        --no-install \
-        --cmake-args ${enable_gazebo} ${build_loc_rviz_plugins} ${extra_opts} -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    if [[ "${ROS_VERSION}" == "1" ]]; then
+        catkin init
+        enable_gazebo=" -DENABLE_GAZEBO=on"
+
+        # Add our cmake to paths and bashrc
+        grep -qF 'source /opt/ros/'$ros_version'/setup.bash' ~/.bashrc || echo 'source /opt/ros/'$ros_version'/setup.bash' >> ~/.bashrc
+        cmake_astrobee_path=`catkin locate -s`/cmake
+        grep -qF ${cmake_astrobee_path} ~/.bashrc || {
+        echo -e '\n' >> ~/.bashrc
+        echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.bashrc
+        }
+        source ~/.bashrc
+
+        shell="$SHELL"
+        if [[ ${shell}  == *"zsh"* ]]; then
+            echo "Setting .zshrc with environment variables..."
+            grep -qF 'source /opt/ros/'$ros_version'/setup.zsh' ~/.zshrc || echo 'source /opt/ros/'$ros_version'/setup.zsh' >> ~/.zshrc
+            grep -qF ${cmake_astrobee_path} ~/.zshrc || {
+                echo -e '\n' >> ~/.zshrc
+                echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.zshrc
+            }
+        fi
 
+        catkin profile add ${profile:-native}
+        catkin profile set ${profile:-native}
+        catkin config --no-extend \
+            --build-space ${workspace_path}build \
+            --install-space ${install_path:-${workspace_path}install} \
+            --devel-space ${workspace_path}devel \
+            --log-space ${workspace_path}log \
+            --no-install \
+            --cmake-args ${enable_gazebo} ${build_loc_rviz_plugins} ${extra_opts} -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    fi
```

Comment:
Makes sense to avoid an error in the ROS2 case, I guess. Do you think it's worth adding an else branch for ROS2 that prints a warning that the rest of the script still needs to be migrated?

Reply:
I checked quickly if colcon had similar configuration, but nothing I could find yet. Let's keep it cleaner and then we check if we need it